src/Forms/ElementalAreaField.php

```diff
@@ -115,11 +116,15 @@ public function getSchemaDataDefaults()
 
         $blockTypes = [];
 
+        /** @var ElementTabProvider $tabProvider */
+        $tabProvider = Injector::inst()->get(ElementTabProvider::class);
```

Comment:
You could use injector to provide this as a service and add the relevant getter/setter for it

Reply:
Ok, that's fine then - just add a comment